app/views/gobierto_admin/shared/_admin_widget_attachment.html.erb

```diff
@@ -116,12 +116,12 @@
 
 <script type="text/x-template" id="file-list-template">
   <small>
-    <a @click.prevent="showFiles = !showFiles" href="#" class="tool_title show-files">
-      <i :class="[showFiles ? 'fa-caret-down' : 'fa-caret-right']" class="fa p_h_r_0_25 soft" aria-hidden="true" v-if="attachments.length"></i>
+    <a @click.prevent="showFiles = !showFiles" @click="toggleList" href="#" class="tool_title show-files">
```

Comment:
Could you toggle the variable in `toggleList` method and avoid two click handlers?